src/doc/grammar.md

```diff
@@ -510,8 +510,8 @@ unit_expr : "()" ;
 ### Structure expressions
 
 ```antlr
-struct_expr : expr_path '{' ident ':' expr
-                      [ ',' ident ':' expr ] *
+struct_expr : expr_path '{' ident [ ':' expr ] ?
+                      [ ',' [ ident [ ':' expr ] ? ] *
```

Comment:
Ah yeah, I'm toying with alacritty and it messes up my brackets in vim currenty, but I obvisouly should have noticed that, my bad.

Should be fixed now.